### DISCUSSION TITLE
[FF7] 60 FPS Bug - Unable to move Cloud in Sector 6 after the cutscene

### DISCUSSION THREAD

**Discussion by nicola36631** [julianxhokaxhiu/FFNx#430] [Q&A] [answered]
**Description:**
In Sector 6 during the cutscene where Tifa is briefly shown and Aeris chases after her, the last dialogue doesn't play out and the player is stuck on top of the slide with inputs disabled (unable to move cloud, open the menu, etc):
![unknown](https://user-images.githubusercontent.com/25088921/161392590-b3f377bf-4351-4ccf-8796-5eea6c7b2d5b.png)
Expected result (with 60fps mod off, while everything else is still on):
The cutsene finishes playing with the last dialogue box and then Cloud jumps off the slide:
![unknown2](https://user-images.githubusercontent.com/25088921/161392634-caa6c481-9eec-45d8-ad50-e214d635bcdf.png)

**Repro steps:**
1. Get to Sector 6
2. Go to where the cutscene normally plays out
3. Get stuck on the slide

**System info:**
OS Name:                   Microsoft Windows 10 Pro
OS Version:                10.0.19044 N/A Build 19044
CPU:                           i5 4670k
GPU:                           NVDIA GTX 1080
RAM:                          16GB DDR3 1886MHz
Monitor:                      2160p@60fps

**Mod managers info:**
7th heaven mod manager: 2.3.3.20
FFNx: 1.10.1.197+

**Enabled Mods info:**
[Tsunamods] Cosmo Memory: 1.011
60 FPS Gameplay (BETA): 1.04
[Tsunamods] Cosmo Memory: 1.011
[Tsunamods] Arranged Soundtrack: 1.21
[Tsunamods] SYW V5 Battle Textures: 5.00
[Tsunamods] SYW V5 Field Textures: 5.21
[Tsunamods] SYW V5 Menu and Fonts: 5.25
[Tsunamods] SYW V5 Minigames Textures: 5.21
[Tsunamods] SYW V5 Spell Textures: 5.00
[Tsunamods] SYW V5 Worldmap Textures: 5.01
[Tsunamods] SYW V6 FMV (30fps): 6.35

**Comment by julianxhokaxhiu** [comment]
Please consider using the already open issue instead of opening a new one the next time: https://github.com/julianxhokaxhiu/FFNx/issues/347

**Comment by julianxhokaxhiu** [comment, marked as answer]
As @tangtang95 reported on #347, consider updating your FFNx to the latest canary version and try again. Thank you.
